### PR TITLE
App template for basic Icecast monitoring

### PR DESCRIPTION
**Pull request by paraenggu**
This PR adds basic Icecast server monitoring.

### CODE REVIEW

File: app/Icecast/doc/README.head.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+Basic monitoring of the [Icecast](http://www.icecast.org).
```
Comment (hairmare) [inline app/Icecast/doc/README.head.md]:
```suggestion
Basic monitoring of [Icecast](http://www.icecast.org).
```

File: app/Icecast/doc/README.head.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+
+## Usage
```
Comment (hairmare) [inline app/Icecast/doc/README.head.md]: Consider moving the usage section to the template.

File: app/Icecast/doc/README.head.md (hunk 1 above)
Comment (paraenggu) [reply app/Icecast/doc/README.head.md]: Fixed.

File: app/Icecast/doc/README.head.md (hunk 2 above)
Comment (paraenggu) [reply app/Icecast/doc/README.head.md]: Removed and updated with the help of the new default usage generation (#74).

**Comment by hairmare** [comment]
Do we want to merge https://github.com/hairmare/zabbix-icecast at some point?

**Pull request merged**